streams/service/src/main/java/com/hortonworks/streamline/streams/service/UDFCatalogResource.java

```diff
@@ -42,44 +42,19 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import javax.ws.rs.Consumes;
+import javax.ws.rs.*;
```

Comment:
Hi @arunmahadevan , thanks for the review.

As noted with @HeartSaVioR 's comments, I will remove all wildcard imports.